openshift/registry.Dockerfile

```diff
@@ -11,6 +11,7 @@ USER 1001
 COPY --from=builder /build/registry /registry
 COPY --from=builder /build/push /push
 COPY openshift/manifests /openshift/manifests
+COPY testdata/images /images
```

Comment:
I think this file contributes to ose-operator-registry IIRC which serves as the foundation for all current RH catalog images.  
I don't think that we should be introducing test data to all catalog images.